src/optable.ts

```diff
@@ -465,6 +465,16 @@ optable[opcodes.STORE_NAME] = function(f: Py_FrameObject) {
     f.locals[name.toString()] = val;
 }
 
+optable[opcodes.UNPACK_SEQUENCE] = function(f: Py_FrameObject) {
+    var val = f.pop();
```

Comment:
Can you add an `assert(val.__getitem__ !== undefined, "Expected a list or tuple type.");`?

From the [CPython source](https://github.com/python/cpython/blob/master/Python/ceval.c#L2096), it looks like this opcode can also receive an Iterable (which only defines the `__iter__` method). I'd like for us to throw an assertion if we encounter this in a test.